.gitmodules

```diff
@@ -1,11 +1,11 @@
 [submodule "lkl"]
 	path = lkl
-	url = ssh://git@github.com/lsds/lkl.git
+	url = https://github.com/lsds/lkl.git
```

Comment:
Can you do the relative path thing from the previous version?  This version now means that it's annoying for people who want to commit changes to any of the submodules because they will be cloned from the read-only (or write-horribly-insecurely) path.

Reply:
I don't know to be honest. I think it's quite standard to use full https URLs for submodules. And if you prefer to use ssh then you can tell git to do that: https://stackoverflow.com/a/31185110. Would that not work? I'm not a big fan of using relative paths in the end because then anyone who forks wouldn't be able to clone --recursive if they haven't forked all submodules as well.

Reply:
So, I feel stupid now.  I forgot that my `.gitconfig` already contained this section:

```ini
[url "ssh://git@github.com"]
        insteadOf = https://github.com
```